### ISSUE TITLE
Dashboards  out of date?

### ISSUE THREAD

**Issue by jlewi** [cncf/devstats-example#7]
I think the dasbhoards for homebrew might be out of date

https://github.com/cncf/devstats-example/blob/master/grafana/dashboards/homebrew/project-statistics.json

Its using

```
          "query": "SELECT \"name\", \"value\" FROM \"project_stats_[[repogroup]]_[[period]]\" WHERE $timeFilter",
          "rawQuery": true,
          "rawSql": "select name, value from \"spstat[[repogroup]]\" where period = '[[period]]'",
```

Comparing to https://github.com/cncf/devstats/blob/master/grafana/dashboards/linux/project-statistics.json and based on conversation in slack I don't think it should be `spstat[[reprogroup]]`

**Comment by lukaszgryglicki** [comment]
No, because it is not merging into a single series in `devstats-example`'s `metrics/shared/metrics.yaml`:
```
  - name: Project stats
    histogram: true
    annotations_ranges: true
    series_name_or_func: multi_row_single_column
    sql: project_stats
```
There is no `merge_series: pstat` line (like in the new devstats).

Devstats uses `"spstat"` table but also ```  - name: Project statistics
    histogram: true
    annotations_ranges: true
    series_name_or_func: multi_row_single_column
    sql: project_stats
    merge_series: pstat```
Definintions are a bit different (`devstats-example` not updated to new) but both of them are correct. `devstats` is newer and just used `merge_series: pstat` just to have lower number of time-series tables - `devstats-example` wasn't updated, but its definition is also valid.

**Comment by lukaszgryglicki** [comment]
Actually, this is the only such dashboard, I will update it then to avoid confusion - give me a few minutes.

**Comment by lukaszgryglicki** [comment]
Fixed by 779994425c94a32f720cc06b2d9b60902251a170
